pkg/network/node/ovscontroller.go

```diff
@@ -283,33 +283,56 @@ func (oc *ovsController) SetUpPod(hostVeth, podIP, podMAC, sandboxID string, vni
 	if err != nil {
 		return -1, err
 	}
-	ofport, err := oc.ensureOvsPort(hostVeth)
+	ofport, err := oc.ensureOvsPort(hostVeth, sandboxID)
 	if err != nil {
 		return -1, err
 	}
 	return ofport, oc.setupPodFlows(ofport, podIP, podMAC, note, vnid)
 }
 
-func (oc *ovsController) SetPodBandwidth(hostVeth string, ingressBPS, egressBPS int64) error {
-	// note pod ingress == OVS egress and vice versa
+func (oc *ovsController) getPortsForSandbox(sandboxID string) ([]string, error) {
+	return oc.ovs.Find("port", "external-ids=sandbox="+sandboxID)
```

Comment:
That does a full match on external-ids (ie, it will fail if external-ids also contains other keys). Changing the `=` between `external-ids` and `sandbox` to a `:` will change it to a subkey match (checking only that external-ids contains a matching sandbox key)

Reply:
+1